### PR TITLE
tests: add JA4 test cases -- v2

### PR DESCRIPTION
**Pull request by satta**
Previous PR: #1407

This PR introduces test cases for TLS and QUIC JA4 calculation.

Changes:
* Added READMEs with some background information on the contents and origin of the pcaps.

## Ticket

Redmine ticket: https://redmine.openinfosecfoundation.org/issues/6379

### CODE REVIEW

**Comment by catenacyber** [comment]
Tests only pass because Suricata PR with the new file is not merged yet

File: tests/ja4-rules/suricata.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+            extended: yes
+        - quic
+
+app-layer:
+  protocols:
+    tls:
+      enabled: yes
+      detection-ports:
+        dp: 443
+      ja4-fingerprints: yes
```
Comment (catenacyber) [inline tests/ja4-rules/suricata.yaml:26]: If we just need this, it can be a command line argument `--set`

File: tests/ja4-tls-quic/README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+### TCP TLS
+
+`t13d1516h2_8daaf6152771_e5627efa2ab1` which means
+
+* `t`: TCP
+* `13`: TLS 1.2
+* `d`: SNI is set
+* `15`: 15 cipher suites in Client Hello
+* `15`: 16 extensions in Client Hello
```
Comment (catenacyber) [inline tests/ja4-tls-quic/README.md:21]: 15 or 16 ?

Comment (satta) [reply tests/ja4-tls-quic/README.md:21]: 16, just a typo.

File: tests/ja4-rules/suricata.yaml (hunk 1 above)
Comment (satta) [reply tests/ja4-rules/suricata.yaml:26]: True. Having a `suricata.yaml` also allows me to control the types that go into the EVE etc. It gives me a better overview of what is done apart from the defaults. If you insist I can change it, but I didn't find it too intrusive to just include a full config file.

Comment (catenacyber) [reply tests/ja4-rules/suricata.yaml:26]: ok for me

**Comment by satta** [comment]
Next PR: #1411

**Pull request closed without merging**